src/app/component/comments/components/add-comment/add-comment.component.html

```diff
@@ -1,15 +1,17 @@
-<form 
-    *ngIf="isLoggedIn"
-    class="wrapper"
-    [formGroup]="addCommentForm" 
-    (ngSubmit)="onSubmit()">
-    <img [src]=avatarImage alt="avatar">
-    <textarea placeholder="Add a comment"
+<div class="container">
+  <app-comment-counter [totalElements]="totalElements"></app-comment-counter>
+</div>
+<form
```

Comment:
Please, write first attribute in the same line as tag and all the rest under it

Reply:
Fixed, but wasn`t my code